lms/djangoapps/courseware/tests/factories.py

```diff
@@ -1,131 +1,30 @@
-# Factories are self documenting  # lint-amnesty, pylint: disable=missing-module-docstring
-
-
+"""
+Build courseware-centric test factories
+"""
 import json
 from functools import partial
 
 import factory
-from django.test.client import RequestFactory
 from factory.django import DjangoModelFactory
 from opaque_keys.edx.keys import CourseKey
 from opaque_keys.edx.locator import CourseLocator
 
+from common.djangoapps.student.tests.factories import UserFactory
 from lms.djangoapps.courseware.models import (
     StudentModule,
     XModuleStudentInfoField,
     XModuleStudentPrefsField,
     XModuleUserStateSummaryField
 )
-from common.djangoapps.student.roles import (
-    CourseBetaTesterRole,
-    CourseInstructorRole,
-    CourseStaffRole,
-    GlobalStaff,
-    OrgInstructorRole,
-    OrgStaffRole
-)
-# Imported to re-export
-from common.djangoapps.student.tests.factories import UserFactory  # Imported to re-export
-from common.djangoapps.student.tests.factories import UserProfileFactory as StudentUserProfileFactory
-
-# TODO fix this (course_id and location are invalid names as constants, and course_id should really be COURSE_KEY)
```

Comment:
@sarina You're welcome! JK, I think you added the comment (thank you!), not necessarily the underlying code. The most recent commit `blame`s you, but I traced it back for a few commits (beyond yours) before giving up.
Anyway, `s/TODO/DONE/`.